### PR TITLE
Fixed public project creation flag in API creation #246

### PR DESCRIPTION
**Pull request by JoseAntonioTorre**
Solved #246 via config check.

### CODE REVIEW

**Comment by almet** [comment]
That seems good to me. Can you add a test for this?

**Comment by JoseAntonioTorre** [comment]
> That seems good to me. Can you add a test for this?

Yes, I will try it!

**Comment by JoseAntonioTorre** [comment]
Ready!

File: ihatemoney/tests/tests.py [hunk 1]
```diff
@@ -281,6 +281,46 @@ def test_project_creation(self):
...
+                    "id": "raclette",
+                    "password": "party",
+                    "contact_email": "raclette@notmyidea.org",
+                },
+            )
+
+            # session is not updated
+            self.assertNotIn("raclette", session)
+
+            # project is created
```
Comment (almet) [inline ihatemoney/tests/tests.py]: I believe the comment should state « not created ».

Comment (JoseAntonioTorre) [reply ihatemoney/tests/tests.py]: uops, you are right!, I change it and upload the commit.

Comment (JoseAntonioTorre) [reply ihatemoney/tests/tests.py]: ready

**Comment by almet** [comment]
Thanks!

**Pull request merged**